pom.xml

```diff
@@ -42,6 +42,8 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <jenkins.version>1.651.2</jenkins.version>
```

Comment:
Maybe I'm not following. This plugin version (when released) is not going to be offered in UC to cores < 1.651.2, so it's granted that `getAllParameters` is there, no? Under what situation `ParametersAction.class.getMethod("getAllParameters")` is going to fail?